server/connectors/slack_connector/client.py

```
@@ -212,7 +254,7 @@ def create_incidents_channel(access_token: str, team_name: str, installer_user_i
         return {"ok": False, "error": "Failed to resolve incidents channel"}
 
     except Exception as e:
-        logger.error(f"Failed to create incidents channel: {e}", exc_info=True)
+        logger.exception(f"[{team_name}] Failed to create incidents channel: {e}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)